#2020/01/30 Patch has not yet been accepted upstream
Description: Default on armv7 for Ubuntu, since gcc-9 deprecates armv5
Author: Gianfranco Costamagna <locutusofborg@debian.org>
Last-Update: 2019-07-10

https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=931795

--- sbcl-1.5.4.orig/src/runtime/Config.arm-linux
+++ sbcl-1.5.4/src/runtime/Config.arm-linux
@@ -9,7 +9,7 @@
 # provided with absolutely no warranty. See the COPYING and CREDITS
 # files for more information.

-CFLAGS += -marm -march=armv5
+CFLAGS += -marm -march=armv7-a
 NM = ./linux-nm

 ASSEM_SRC = arm-assem.S ldso-stubs.S
